arm-advisor/2016-07-12-preview/swagger/advisor.json

```diff
@@ -0,0 +1,461 @@
+				"consumes": [],
+				"produces": ["application/json",
+				"text/json"],
+				"parameters": [{
+					"name": "subscriptionId",
+					"in": "path",
+					"required": true,
+					"type": "string",
+					"format": "uuid"
+				},
+				{
+					"name": "api-version",
+					"in": "query",
+					"required": true,
+					"type": "string"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"type": "object"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		}
+	},
+	"definitions": {
+		"ResourceRecommendationDetail": {
+			"type": "object",
+			"properties": {
+				"actions": {
+					"type": "array",
+					"items": {
+						"$ref": "#/definitions/ResourceRecomendationAction"
+					}
+				},
+				"description": {
+					"type": "string"
+				},
+				"label": {
+					"type": "string"
+				},
+				"learnMoreLink": {
+					"type": "string"
+				},
+				"category": {
+					"enum": ["HighAvailability",
+					"Security",
+					"Performance",
+					"Cost"],
+					"type": "string"
+				},
+				"id": {
+					"type": "string"
+				},
+				"impact": {
+					"enum": ["High",
+					"Medium",
+					"Low"],
+					"type": "string"
+				},
+				"impactedName": {
+					"type": "string"
+				},
+				"impactedType": {
+					"type": "string"
+				},
+				"lastUpdated": {
+					"format": "date-time",
+					"type": "string"
+				},
+				"shortDescription": {
+					"$ref": "#/definitions/ShortDescription"
+				},
+				"subscriptionId": {
+					"format": "uuid",
+					"type": "string"
+				}
+			}
+		},
+		"ResourceRecomendationAction": {
+			"type": "object",
+			"properties": {
+				"actionType": {
+					"enum": ["Blade",
+					"Document",
+					"PowerShell",
+					"CLI"],
+					"type": "string"
+				},
+				"caption": {
+					"type": "string"
+				},
+				"description": {
+					"type": "string"
+				},
+				"metadata": {
+					"type": "object",
+					"additionalProperties": {
+						"type": "object"
+					}
+				}
+			}
+		},
+		"ShortDescription": {
+			"type": "object",
+			"properties": {
+				"problem": {
+					"type": "string"
+				},
+				"solution": {
+					"type": "string"
+				}
+			}
+		},
+		"CollectionResponse[OperationEntity]": {
+			"type": "object",
+			"properties": {
+				"nextLink": {
+					"type": "string"
+				},
+				"value": {
+					"type": "array",
+					"items": {
+						"$ref": "#/definitions/OperationEntity"
+					}
+				}
+			}
+		},
+		"OperationEntity": {
+			"type": "object",
+			"properties": {
+				"name": {
+					"type": "string"
+				},
+				"display": {
+					"$ref": "#/definitions/OperationDisplayInfo"
+				}
+			}
+		},
+		"OperationDisplayInfo": {
+			"type": "object",
+			"properties": {
+				"description": {
+					"type": "string"
+				},
+				"operation": {
+					"type": "string"
+				},
+				"provider": {
+					"type": "string"
+				},
+				"resource": {
+					"type": "string"
+				}
+			}
+		},
+		"SubscriptionNotificationDefinition": {
+			"type": "object",
+			"properties": {
```

Comment:
"Consider using [x-ms-client-flatten](https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-client-flatten) to provide a better end user experience"